### DISCUSSION TITLE
error [hello] unexpected packet [72] from server

### DISCUSSION THREAD

**Discussion by tabwizard** [Altinity/clickhouse-backup#532] [Q&A] [answered]
Здравствуйте,
пытаюсь подключиться к "Managed Service for ClickHouse" в YandexCloud, на 
`>clickhouse-backup tables`
получаю
`error [hello] unexpected packet [72] from server`
пытался подключаться как с помощью локально скачанного файла, так и через docker:
```bash
docker run --rm -it --network host -v "/var/lib/clickhouse:/var/lib/clickhouse" \
-e REMOTE_STORAGE="none" \
-e CLICKHOUSE_USERNAME="my_user" \
-e CLICKHOUSE_PASSWORD="XXXXXXXXXXXXXXXXXXXX" \
-e CLICKHOUSE_HOST="c-c9qjuhbubguyvh95qgt.rw.mdb.yandexcloud.net" \
-e CLICKHOUSE_PORT="8123" \
alexakulov/clickhouse-backup tables
2022/10/13 05:55:49.173035 error [hello] unexpected packet [72] from server
```
Помогите пожалуйста

**Comment by Slach** [comment, marked as answer]
Лучше используйте встроенный в Yandex Cloud backup
https://cloud.yandex.ru/docs/managed-clickhouse/operations/cluster-backups

clickhouse-backup в Yandex.Cloud работать нормально не будет, потому что ему требуется чтобы был прямой доступ к дискам при upload
максимум что получится сделать - локальные бекапы, или бекапы схемы \ конфигов \ rbac
remote бекапы можно организовать через скриптинг 
remote_storage: custom

если очень хочется извращений
почитайте внимательно, https://cloud.yandex.ru/docs/managed-clickhouse/operations/connect
CLICKHOUSE_PORT=9000 
попробуйте если изнутри Yandex Cloud соединятетесь

если снаружи тогда CLICKHOUSE_PORT=9440
нужен TLS CA 
https://cloud.yandex.ru/docs/managed-clickhouse/operations/connect#get-ssl-cert

добавьте `-v ca.pem:/etc/clickhouse-backup/ca.pem -e CLICKHOUSE_TLS_CA=/etc/clickhouse-backup/ca.pem`

**Reply by Slach** [reply]
перечитайте еще раз мой ответ, лучше два или три раза
